code/script.js

```diff
@@ -244,100 +296,155 @@ const generateBoard = () => {
   })
 }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
+// Randomly select a person from the characters array,
+// and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
-// This function to start (and restart) the game
+// Start & initialize the game by
+// bringing in the characters from the array,
+// (re)set the board,
+// (re)set the guess list,
+// invoke the generation of characters,
+// and invoke the selection of secret character
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  board.style.display = 'flex'
+  winOrLose.style.display = 'none'
+  questions.selectedIndex = '0'
```

Comment:
I'm not sure what this is doing here, maybe you wanted to implement a counter that keeps track of the number of guesses? In that case, you could declare a global variable and update the value of it every time the checkMyGuess is invoked or similar :)

Reply:
`board.style.display = 'flex'` makes sure the `board` section is being displayed upon start, basically reverses what the `checkMyGuess`  function does in order to display the `winOrLose` section.
`winOrLose.style.display = 'none'` sets the `winOrLose` section back to its original state of `none`.

Whereas `questions.selectedIndex = '0'` just resets the drop-down selection to the first option (`disabled selected`), instead of displaying the last guess in the drop-down.

I tried various `location.reload()`-esque solutions, but neither worked as I wanted them to... At my current level this was what I came up with. :) No plan for any counter, but that could be a cool addition to track the score or how many guesses were made on average to reach the correct answer!

Reply:
Ahh of course 🤦‍♀️  You sure have attention to detail! 💯